instagram-app/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Check your errors.

Compiled with warnings.

./src/components/PostContainer/PostsPage.js
  Line 24:  Expected to return a value at the end of arrow function  array-callback-return

Reply:
Error is still there. Proper way to write it would be

```
  filterSearch = (e) => {
    e.preventDefault();
    const filtered = this.state.posts.filter(post => post.username.includes(e.target.value))
    
    this.setState({ filteredPosts: filtered });
  };
```